tools/lkl/lib/hijack/hijack.c

```diff
@@ -292,6 +305,24 @@ int select(int nfds, fd_set *r, fd_set *w, fd_set *e, struct timeval *t)
 }
 
 HOOK_CALL_USE_HOST_BEFORE_START(epoll_create);
+int epoll_create(int flags)
+{
+	int nRes;
```

Comment:
Please don't use camelCase variable and function name, to stay consistent with the Linux kernel coding style. For exaple, you can use `res` or `ret`.  Use a newline to separate variable definition from the function code.